azure-quantum/azure/quantum/optimization/problem.py

```diff
@@ -212,3 +218,16 @@ def evaluate(self, configuration: Union[Dict[int, int], Dict[str, int]]) -> floa
                 total_cost += term.evaluate(configuration_transformed)
 
         return total_cost
+
+    def is_large(self) -> bool:
+        """Determines if the current problem is large. 
+        "large" is an arbitrary threshold and can be easily changed. Based on usage data, we have defined a 
+        large problem to be 2500+ variables AND 1mil+ terms. 
```

Comment:
can we refer the constants (`NUM_VARIABLES_LARGE ` and `NUM_TERMS_LARGE `) here ?

also, can we spell out `1mil+` to `1million+`, as in some contexts `mil` can mean `thousand`

Reply:
done